composer.json

```diff
@@ -6,8 +6,10 @@
 		"php": ">=7.1",
 		"doctrine/dbal": "^2.5",
 		"doctrine/orm": "^2.5",
+		"doctrine/migrations": "~1.8",
 		"gedmo/doctrine-extensions": "^2.4",
-		"ramsey/uuid": "^3.7"
+		"ramsey/uuid": "^3.7",
+		"symfony/yaml": "~4.1"
```

Comment:
Is the YAML library strictly necessary or won't it be installed anyway implied doctrine-migrations?

Reply:
Unfortunately it's not a sub-dependency since using YAML is considered an optional feature by doctrine-migrations, I guess.